git_notarize.py

```diff
@@ -153,34 +198,33 @@ def get_debranded_imports_tag(self, tag: str):
         # Without AlmaLinux custom suffixes (.alma, .alma.2), if any
         debranded_nvr = re.sub(r"\.alma.*$", "", tag_nvr)
 
-        debranded_tag = (
-            debranded_type,
-            debranded_distro,
-            debranded_nvr
-        )
+        debranded_tag = (debranded_type, debranded_distro, debranded_nvr)
 
         return "/".join(debranded_tag)
 
+
 def notarize(
-        cw: CasWrapper,
-        repo_path: str,
-        upstream_commit_sbom_hash: str = None
-    ):
+    immudb_wrapper: ImmudbWrapper,
+    repo_path: str,
+    upstream_commit_sbom_hash: str = None,
+):
     metadata = {
         "sbom_api_ver": "0.2",
     }
 
     if upstream_commit_sbom_hash:
         logging.info(
-            "Notarizing AlmaLinux source using the " \
-            "upstream_commit_sbom_hash %s" % upstream_commit_sbom_hash
+            "Notarizing AlmaLinux source using the "
+            "upstream_commit_sbom_hash %s",
+            upstream_commit_sbom_hash,
         )
         metadata['upstream_commit_sbom_hash'] = upstream_commit_sbom_hash
 
-    notarized_hash = cw.notarize(
-        local_path=f"git://{repo_path}",
-        metadata=metadata
+    result = immudb_wrapper.notarize_git_repo(
+        repo_path=repo_path,
+        user_metadata=metadata,
     )
+    notarized_hash = result.get('value', {}).get('Hash')
```

Comment:
Do you return the data in the same format as CAS CLI returns?

Reply:
Yes, here is an example of `immudb_wrapper` output
```
In [3]: cli.authenticate('b725b285db9d4871b9aac848cf5a4e38bbb228acf33c10dffb4adfae54f7a6a1')
Out[3]: 
{'id': 8,
 'key': 'b725b285db9d4871b9aac848cf5a4e38bbb228acf33c10dffb4adfae54f7a6a1',
 'value': {'Name': 'chan-0.0.4-3.el8.src.rpm',
  'Kind': 'file',
  'Size': '32.96 KB',
  'Hash': 'b725b285db9d4871b9aac848cf5a4e38bbb228acf33c10dffb4adfae54f7a6a1',
  'Signer': 'immudb',
  'Metadata': {'sbom_api_ver': '0.2',
   'build_id': 37,
   'build_host': '08e82fea6dbb',
   'build_arch': 'x86_64',
   'built_by': 'anfimovdm <anfimovdan@gmail.com>',
   'source_type': 'git',
   'git_url': 'https://git.almalinux.org/rpms/chan.git',
   'git_ref': 'c8',
   'git_commit': '08ac60d5244114259312eaca770568f47a567217',
   'name': 'chan',
   'epoch': None,
   'version': '0.0.4',
   'release': '3.el8',
   'arch': 'x86_64',
   'sourcerpm': None}},
 'timestamp': 1693302574,
 'verified': True,
 'refkey': None,
 'revision': 1}
```